PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/tree/TreeManager.java

```diff
@@ -232,14 +235,12 @@ private void initializeGraphSizeListeners() {
    * @param root the root of the part of the tree which should be shown.
    */
   @TestId(id = "setRoot")
-  private void setRoot(IPhylogeneticTreeNode root) {
+  public void setRoot(IPhylogeneticTreeNode root) {
```

Comment:
Probably want to rename this to `setCurrentRoot`. I was wondering why in the constructor you do `rootNode = ` and then call this method.